src/radio-tile/radio-tile.scss

```diff
@@ -59,7 +58,7 @@
   user-select: none;
   // Create stacking context
   position: relative;
-  z-index: 0;
+  isolation: isolate;
```

Comment:
If there's isolation: isolate, then none of the other z-indexes in this file need to be changed